src/content/basics/automatic-termination-of-bad-nodes/index.md

```diff
@@ -10,12 +10,14 @@ owner:
 
 # Automatic termination of unhealthy nodes
 
+{{< platform_support_table aws="ga=v12.6.0" azure="ga=v13.1.0" kvm="roadmap=https://github.com/giantswarm/roadmap/issues/176" >}}
```

Comment:
Currently clarifying whether this is already implemented. But that should not block this PR.